OpenLiberty/guide-maven-multimodules
# Dev mode: simplified pom

- use the ideal(simplified) pom.xml 
- update the Readme content to use dev mode - N/A
  - keep the flow same (let discuss if the flow not work)
- convert `*Test` class to `*IT`
- convert to use JUnit 5 
  - use JUnit5 test order if need
- make sure to correctly configure the ports in test classes, server.xml, and pom.xml
- update travis test
   - `mvn install` cannot be used to verify anymore
